src/contexts/useEmbalmer.ts

```diff
@@ -0,0 +1,49 @@
+// import { ethers } from 'ethers';
+import { useState } from 'react';
+import { useAccount, useContractRead, useContract, useProvider, useSigner } from 'wagmi';
+import { ViewStateFacet__factory, EmbalmerFacet__factory } from '../assets/typechain';
+import { ISarcophagus } from '../types/sarcophagi.interfaces';
+import useArchaeologistService from '../contexts/useArchaeologistService';
+
+const useEmbalmer = () => {
+  const { address: embalmerAddress } = useAccount();
+  const provider = useProvider();
+  const signer = useSigner();
+  const [sarcophagi, setSarcophagi] = useState<ISarcophagus[]>([]);
+  const { getConfirmations } = useArchaeologistService();
+
+  const viewStateContract = useContract({
+    addressOrName: process.env.REACT_APP_LOCAL_CONTRACT_ADDRESS || '',
+    contractInterface: ViewStateFacet__factory.abi,
+    signerOrProvider: provider,
+  });
+
+  const getEmbalmersarcophagi = useContractRead({
+    addressOrName: process.env.REACT_APP_LOCAL_CONTRACT_ADDRESS || '',
+    contractInterface: ViewStateFacet__factory.abi,
+    functionName: 'getEmbalmersarcophagi',
+    args: [embalmerAddress],
+  });
+
+  async function updateSarcophagi() {
```

Comment:
Can you remove the console logs in this function?

Reply:
rookie error ... removed